zapcore/field.go

```diff
@@ -106,6 +106,7 @@ type Field struct {
 	Type      FieldType
 	Integer   int64
 	String    string
+	Level     *Level
```

Comment:
Making this attribute a pointer seemed the cleanest way to implement this - doing so would make all existing fields' `Level` attribute be `nil`.